### DISCUSSION TITLE
How do you change the width?

### DISCUSSION THREAD

**Discussion by skyisthela** [Flow-Launcher/Flow.Launcher#3016] [Q&A] [answered]
I know this is a dumb question but I can't find an option to change it in the settings, only the height. I couldn't find anyone else asking either

**Comment by bytenekos** [comment, marked as answer]
Should be able to just drag it from the edges like so:


https://github.com/user-attachments/assets/c8405445-d666-4ae0-9907-651a0d8af4cb

**Reply by skyisthela** [reply]
i can't believe this never even occurred to me thanks
